### PR TITLE
environment TRACEFOSS-2373 stable environments preparation

### PR DESCRIPTION
**Pull request by ds-ext-sceronik**

### CODE REVIEW

**Review by ds-mwesener — CHANGES_REQUESTED** [review]
Questions: 
Do we also need to provide data? If yes we would need an edc provider.

File: charts/connector/aasregistry/values-stable-a.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        limits:
+          cpu: 400m
+          memory: 1Gi
+        requests:
+          cpu: 200m
+          memory: 1Gi
+  registry:
+    authentication: false
+    idpIssuerUri: https://centralidp.stable.demo.catena-x.net/auth/realms/CX-Central  # check centralidp url when provided
+    host: trace-x-registry-stable-a.stable.demo.catena-x.net
```
Comment (ds-mwesener) [inline charts/connector/aasregistry/values-stable-a.yaml:37]: Want to go with tx-registry? To follow our new naming model. This has not been changed on the other environments so far. What do you think?

File: charts/connector/submodelservers/values-stable-a.yaml [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+ingress:
+  enabled: true
+  className: "nginx"
+  annotations:
+    nginx.ingress.kubernetes.io/ssl-passthrough: "false"
+    nginx.ingress.kubernetes.io/backend-protocol: "HTTP"
+    nginx.ingress.kubernetes.io/force-ssl-redirect: "false"
+  hosts:
+    - host: "tracex-submodel-server-stable-a.stable.demo.catena-x.net"
```
Comment (ds-mwesener) [inline charts/connector/submodelservers/values-stable-a.yaml:67]: Here as well? tx-submodel-server...

File: charts/connector/submodelservers/values-stable-a.yaml [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
+  tls:
+    - hosts:
+        - "tracex-submodel-server-stable-a.stable.demo.catena-x.net"
```
Comment (ds-mwesener) [inline charts/connector/submodelservers/values-stable-a.yaml:73]: .

File: charts/connector/submodelservers/values-stable-b.yaml [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+ingress:
+  enabled: true
+  className: "nginx"
+  annotations:
+    nginx.ingress.kubernetes.io/ssl-passthrough: "false"
+    nginx.ingress.kubernetes.io/backend-protocol: "HTTP"
+    nginx.ingress.kubernetes.io/force-ssl-redirect: "false"
+  hosts:
+    - host: "tracex-submodel-server-stable-b.stable.demo.catena-x.net"
```
Comment (ds-mwesener) [inline charts/connector/submodelservers/values-stable-b.yaml:67]: .

File: charts/connector/submodelservers/values-stable-b.yaml [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+      paths:
+        - path: /
+          pathType: ImplementationSpecific
+  tls:
+    - hosts:
+        - "tracex-submodel-server-stable-b.stable.demo.catena-x.net"
```
Comment (ds-mwesener) [inline charts/connector/submodelservers/values-stable-b.yaml:73]: .

File: charts/traceability-foss/values-stable-a.yaml [hunk 6]
```diff
@@ -0,0 +1,469 @@
...
+  # @url: https://cloud.google.com/blog/products/containers-kubernetes/kubernetes-best-practices-resource-requests-and-limits
+  resources:
+    limits:
+      cpu: 500m
+      memory: 512Mi
+    requests:
+      cpu: 100m
+      memory: 512Mi
+
+  springprofile: stable
```
Comment (ds-mwesener) [inline charts/traceability-foss/values-stable-a.yaml:96]: As we will not have specific profiles anymore this should be dev right?

File: charts/traceability-foss/values-stable-a.yaml [hunk 7]
```diff
@@ -0,0 +1,469 @@
...
+    baseUrl: "https://semantics.stable.demo.catena-x.net/discoveryfinder/api/administration/connectors/discovery/search"  # check DF url when provided
+
+  config:
+    allowedCorsOriginFirst: "http://localhost:4200/"
+    allowedCorsOriginSecond: "https://traceability-portal-stable-a.stable.demo.catena-x.net/"
+
+  irs:
+    baseUrl: "https://tx-irs-stable-a.stable.demo.catena-x.net"
+  registry:
+    urlWithPath: "https://trace-x-registry-stable-a.stable.demo.catena-x.net/semantics/registry/api/v3.0"
```
Comment (ds-mwesener) [inline charts/traceability-foss/values-stable-a.yaml:146]: If we change - please also change here.

File: charts/traceability-foss/values-stable-a.yaml [hunk 8]
```diff
@@ -0,0 +1,469 @@
...
+          - path: /
+            pathType: ImplementationSpecific
+    tls:
+      - hosts:
+          - "traceability-stable-a.stable.demo.catena-x.net"
+        secretName: tls-secret
+
+  traceability:
+    bpn: "CHANGE_ME"  # PROVIDE PROPER BPN
+    url: "https://traceability-stable-a.stable.demo.catena-x.net"
```
Comment (ds-mwesener) [inline charts/traceability-foss/values-stable-a.yaml:115]: Should be tx- not traceability?

File: charts/traceability-foss/values-stable-a.yaml (hunk 6 above)
Comment (ds-ext-sceronik) [reply charts/traceability-foss/values-stable-a.yaml:96]: will keep only dev

File: charts/connector/edc-provider/values-stable-a.yaml [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+    resources:
+      limits:
+        cpu: 200m
+        memory: 512Mi
+      requests:
+        cpu: 100m
+        memory: 512Mi
+
+  backendService:
+    httpProxyTokenReceiverUrl: "https://traceability-stable-a.stable.demo.catena-x.net/api/callback/endpoint-data-reference"
```
Comment (ds-mwesener) [inline charts/connector/edc-provider/values-stable-a.yaml:66]: Should be the backend url, correct?

File: charts/connector/edc-provider/values-stable-b.yaml [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+    resources:
+      limits:
+        cpu: 200m
+        memory: 512Mi
+      requests:
+        cpu: 100m
+        memory: 512Mi
+
+  backendService:
+    httpProxyTokenReceiverUrl: "https://traceability-stable-b.stable.demo.catena-x.net/api/callback/endpoint-data-reference"
```
Comment (ds-mwesener) [inline charts/connector/edc-provider/values-stable-b.yaml:66]: Should be the backend url.

File: charts/connector/submodelservers/values-stable-a.yaml [hunk 11]
```diff
@@ -0,0 +1,133 @@
...
+  requests:
+    cpu: 100m
+    memory: 512Mi
+
+autoscaling:
+  enabled: false
+  minReplicas: 1
+  maxReplicas: 100
+  targetCPUUtilizationPercentage: 80
+  # targetMemoryUtilizationPercentage: 80
```
Comment (ds-mwesener) [inline charts/connector/submodelservers/values-stable-a.yaml:92]: Remove comment

**Review by ds-mwesener — APPROVED** [review]

**Pull request merged**